libphal/exception.H

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <exception>
+
+namespace libphal
+{
+namespace exception
+{
+
+enum ERR_TYPE {
+	NONE = 0,
+};
+
+typedef std::map<ERR_TYPE, const char *> errMsgMapType;
+const errMsgMapType errMsgMap = {{NONE, ""}};
+
+// libphal specific errors base exception class
+struct LibphalError : public std::exception {
```

Comment:
LibphalError -> LibPHALError? or LibPhalError or PhalError - suggestion

Reply:
phal "lowercase" name is usualy convention used commonly